ci-operator/step-registry/openstack/provision/sriov-networknodepolicy/openstack-provision-sriov-networknodepolicy-commands.sh

```diff
@@ -6,26 +6,26 @@ set -o pipefail
 
 wait_for_sriov_pods() {
     # Wait up to 15 minutes for SNO to be installed
-    for _ in $(seq 1 90); do
+    for _ in $(seq 1 15); do
         SNO_REPLICAS=$(oc get Deployment/sriov-network-operator -n openshift-sriov-network-operator -o jsonpath='{.status.readyReplicas}' || true)
         if [ "${SNO_REPLICAS}" == "1" ]; then
             FOUND_SNO=1
             break
         fi
         echo "Waiting for sriov-network-operator to be installed"
-        sleep 10
+        sleep 60
     done
 
     if [ -n "${FOUND_SNO:-}" ] ; then
         # Wait for the pods to be started from the operator
-        for _ in $(seq 1 24); do
-            NOT_RUNNING_PODS=$(oc get pods --no-headers -n openshift-sriov-network-operator -o jsonpath='{.items[*].status.containerStatuses[*].ready}' | grep false | wc -l || true)
+        for _ in $(seq 1 8); do
```

Comment:
here we reduce the number of retries and the waiting time between retries, is it because it's not worth waiting so long? can you please explain?

Reply:
It was to honor the comment at line 33. Also, the increase of sleeping 60 secs at line 28 was a mistake in my previous commit, it should have been at line 16 instead.